apps/api/.env.example

```diff
@@ -30,6 +30,8 @@ CIRCLE_URL=https://api-sandbox.circle.com
 EMAIL_FROM=
 EMAIL_NAME=AlgoMart
 
+CUSTOMER_SERVICE_EMAIL=
```

Comment:
I think it might be worth adding a comment explaining what this variable does. If I understand correctly, some clients want to their customer service people to receive email notifications and some don't, so `CUSTOMER_SERVICE_EMAIL` is optional, when it's provided they get emails, when it's not, they don't.